docs/api-reference/deck.md

```diff
@@ -147,7 +148,7 @@ Callback arguments:
 
 The `onViewStateChange` callback is fired when the user has interacted with the deck.gl canvas, e.g. using mouse, touch or keyboard.
 
-`onViewStateChanged({viewState})`
+`onViewStateChange({viewState})`
```

Comment:
Nit: Use ```js

Reply:
> Nit: Use ```js

Yes... this compact pattern is used all over the docs, I think it will be better to make a consolidated update than to start changing it in just some places.